### DISCUSSION TITLE
uncheck for the schema

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#141] [General]
Hi there, I found the global state unchecked code in the rgb-core/contract.rs:
<img width="594" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/82d1d7e6-7020-400b-aa2e-5e64a3df7cbc">
So, do we need to implement other state unchecked method ?(For owned_type, transition, and extension defined in Schema)?

**Comment by dr-orlovsky** [comment]
I do not recall that there was any need for that.

Global state is indexed by the state type, while assignments (owned state) is not
